requirements.txt

```diff
@@ -2,9 +2,9 @@ pycryptodome
 envyaml
 tk
 tksheet
-requests
+requests==5.0.25
```

Comment:
should be tksheet==5.0.25

I did push the suggested changes to master, and later realized you have created a PR as well.
thanks for that.